src/common/src/types/interval.rs

```diff
@@ -284,6 +284,18 @@ impl From<&'_ IntervalUnitProto> for IntervalUnit {
     }
 }
 
+impl From<NaiveTimeWrapper> for IntervalUnit {
+    fn from(time: NaiveTimeWrapper) -> Self {
+        let mut ms: i64 = (time.0.num_seconds_from_midnight() * 1000) as i64;
+        ms += (time.0.nanosecond() / 1_000_000) as i64;
```

Comment:
Do we need to care about microsecond precision for `IntervalUnit`?
Seems like this loses the last 6 digits of the nanoseconds.

Reply:
I guess this can be fixed separately, it is to do with the current implementation of `NaiveTimeWrapper` out of scope for this PR.

Reply:
You are right. We need to update `IntervalUnit` from storing milliseconds to microseconds, which is better handled separately.